### PR TITLE
clothesmate pride jackets are no longer contrabrand because not a single one of our players can be trusted

### PR DESCRIPTION
**Pull request by azzzertyy**
# Document the changes in your pull request
as title

# Changelog
:cl:  
tweak: pride jackets are not contraband
/:cl:

### CODE REVIEW

**Review by SapphicOverload — APPROVED** [review]
gay

**Comment by Therandomhoboo** [comment]
hugbox

**Comment by iloveloopers** [comment]
there should be downsides to being gay, clearly

**Comment by Hislyna** [comment]
How did I know this was gonna happen

**Comment by adamsong** [comment]
I'd be interested in an explanation of the downvotes

File: code/modules/vending/clothesmate.dm [hunk 1]
```diff
@@ -185,7 +185,10 @@
 		            /obj/item/clothing/shoes/singery = 1,
 		            /obj/item/clothing/shoes/singerb = 1,
 		            /obj/item/clothing/shoes/yogs/trainers = 1,
-		            /obj/item/clothing/shoes/yogs/fuzzy_slippers = 3) //yogs clothes for clothesmate end here
+		            /obj/item/clothing/shoes/yogs/fuzzy_slippers = 3,
+					/obj/item/clothing/suit/jacket/pride = 3,
+					/obj/item/clothing/suit/jacket/pride/mlm = 2,
+					/obj/item/clothing/suit/jacket/pride/lesbian = 2) //yogs clothes for clothesmate end here
```
Comment (JohnFulpWillard) [inline code/modules/vending/clothesmate.dm:191]: wtf is going on here
is your pr updated to master?
why isnt the suits in contraband on this branch but it is on master? you're not removing it or anything

**Comment by MajManatee** [comment]
Yeah according to this they arent contraband already... This just adds them to the vendor?

**Comment by adamsong** [comment]
> Yeah according to this they arent contraband already... This just adds them to the vendor?

Just confirmed, they are definitely in the contraband list

**Review by adamsong — DISMISSED** [review]
Remove from contraband list

File: code/modules/vending/clothesmate.dm (hunk 1 above)
Comment (azzzertyy) [reply code/modules/vending/clothesmate.dm:191]: weird

**Review by JohnFulpWillard — APPROVED** [review]

**Pull request merged**